### ISSUE TITLE
Issues starting a new tmux session on startup

### ISSUE THREAD

**Issue by sollymay** [romkatv/powerlevel10k#2343]
Hey @romkatv thank you so much for this project. its really cool and I've started using it recently migrating from powerlevel9k.

I've been struggling to make my config work after I set up p10k.zsh

Every time I open my terminal, I want to have a fresh tmux session, however when I use the command for it, my session gets closed. I was using the exact same command in powerlevel9k without any issues (every time I opened a new terminal window or a new terminal altogether, TMUX would open. Now, it gets automatically closed unless I comment out this.

```
case $- in *i*)
    [ -z "$TMUX" ] && exec tmux
esac
```

Do you know why this might be happening and how to avoid it? I tried moving this command to after loading p10k.zsh on my .zshrc file but it didn't work either

**Comment by romkatv** [comment]
You need to place this command at the very top of ~/.zshrc.

**Comment by sollymay** [comment]
yep, that worked 👍🏻. Don't know why I didn't try that before posting 😄. thank you!!!
